### PR TITLE
Feature/setup django

### PR DESCRIPTION
**Pull request by SHGODE**
@mattdrayer 

Opening a WIP pull request.

### CODE REVIEW

File: app/.flake8 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[flake8]
+exclude = 
+    migrations
+    __pycache__,
+    manage.py,
+    settings.py
```
Comment (mattdrayer) [inline app/.flake8:6]: Need a newline

File: app/api/serializers.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                  'provider_name',
+                  'date_inserted',
+                  'metadata_key',
+                  'metadata_hash',
+                  'metadata',
+                  'record_status',
+                  'date_deleted',
+                  'metadata_validation_date',
+                  'metadata_validation_status']
+# class SupplementalLedgerSerializer(serializers.Serializer):
```
Comment (mattdrayer) [inline app/api/serializers.py:21]: If this hidden code is not necessary we should remove it

File: app/api/views.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from api.models import MetadataLedger
+from api.serializers import MetadataLedgerSerializer
+
+logger = logging.getLogger('dict_config_logger')
+
+class MetadataLedgerView(APIView):
+    """
+    List all snippets, or create a new snippet.
+    """
+    # def get(self, request, format=None):
```
Comment (mattdrayer) [inline app/api/views.py:15]: If this hidden code is not necessary we should remove it.

**Review by mattdrayer — APPROVED** [review]
Just a few small suggestions!

**Comment by mattdrayer** [comment]
@SHGODE looks like there are just a couple of remaining code structure items to address, otherwise this branch should be good to rebase on the latest version of `main`, resolve conflicts, squash commits, and merge.  Let me know if you have any further questions about it.

**Comment by SHGODE** [comment]
Hi Matt, We have one more branch **feature/update-record-status** which is merged with **feature/setup-django**. So we are thinking of deleting this branch (feature/setup-django) and merging feature/update-record-status with main. Will it okay with you?

**Comment by mattdrayer** [comment]
If the branch has been merged to this one then it should be safe to delete the original, yes.

**Comment by mattdrayer** [comment]
Wait, it's not clear which branch is the original and which branch will remain.  Pull Request #5 does not have nearly the amount of files in it (12 versus 199).  This seems like a substantial difference?

**Comment by SHGODE** [comment]
**feature/update-record-status** this branch is original, and it has everything in it. So we are planning to deleted **feature/setup-django** this one.

**Comment by mattdrayer** [comment]
Okay, sounds good.

**Pull request closed without merging**